packages/shared/src/utils/validation/form-validations.ts

```diff
@@ -24,26 +31,24 @@ export const filterByLandingCompany = (landing_company, schema) =>
  * @param schema
  * @return {function(*=): {}}
  */
-export const generateValidationFunction = (landing_company, schema) => {
+export const generateValidationFunction = (landing_company: string, schema: TSchema) => {
     const rules_schema = filterByLandingCompany(landing_company, schema);
-    const rules = {};
+    const rules: { [key: string]: any } = {};
     Object.entries(rules_schema).forEach(([key, opts]) => {
         rules[key] = opts.rules;
     });
 
-    return values => {
-        const errors = {};
+    return (values: string[]) => {
+        const errors: { [key: string]: string } = {};
 
         Object.entries(values).forEach(([field_name, value]) => {
             if (field_name in rules) {
-                rules[field_name].some(([rule, message, options]) => {
+                rules[field_name].some(([rule, message, options]: any) => {
```

Comment:
same here!

Reply:
@niloo-fs no need to assert, please remove **: any** here 🙂